packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/add/add.styles.scss

```diff
@@ -0,0 +1,7 @@
+ovh-manager-pci-project-database-namespaces-edit-component {
+  .overflow-collapsible {
+    .oui-collapsible__header[aria-expanded='true'] ~ .oui-collapsible__wrapper {
```

Comment:
**question:** Can you describe the issue here?

IMHO if the behavior is relevant, we should consider to add this rule directly on [ovh/ovh-ui-kit](https://github.com/ovh/ovh-ui-kit).

Reply:
Hi Antoine,
Here's screenshots with: 
![withcss](https://user-images.githubusercontent.com/24541481/154439249-81a82854-63ae-4c7f-9737-9bb11537ae2e.png)
and without fix:
![without](https://user-images.githubusercontent.com/24541481/154439323-58ea2569-e45e-44f8-931c-a3a22c49ad2c.png)

I'm not sure we want to change the overflow rule for every case though.
BTW, there is an error here, it should be ```ovh-manager-pci-project-database-namespaces-add-component```, I'll fix it

Reply:
**question:** Can we avoid code duplication?
see: `packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/edit/edit.styles.scss`